tools/build_config/Jenkinsfile

```diff
@@ -135,25 +135,25 @@ properties([
     ),
     string(
       defaultValue: '3.19',
-      description: 'The version of CMake to run the build with.',
+      description: 'The version of CMake to run the build with. Not relevant on current Windows pipelines.',
```

Comment:
Better to say where it is used than where it is not used.

Reply:
I preferred to say where it wasn't in case new pipelines were added. Can update.

Reply:
Just adding my agreement with Nick. The "Not relevant" looks odd. Maybe it would be better to centralise scopes for these comments in another document rather than piecemeal through the descriptions,

Reply:
Ok, will fix. This is what shows up on the Jenkins "Build with Parameters", so it's documenting it there for user awareness, as well as within the Jenkins script.